src/style.css

```
@@ -1 +1,57 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0,05);
+}
+
+.header__logo {
+  width: 40px;
+  height: 40px;
+}
+
+.header__li {
+  display: inline-block;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  margin-left: 16px;
+  position: relative;
+}
+
+.header__li:first-child {
+  margin-left: 0;
+}
+
+.header__link {
+  color: #000;
+  text-decoration: none;
+  transition: 0.5s;
```

You should explicitly define properties for transition
```suggestion
  transition: color 0.5s;
```